s3tests_boto3/functional/test_s3select.py

```diff
@@ -314,22 +387,147 @@ def test_count_operation():
     bucket_name = "test"
     num_of_rows = 1234
     obj_to_load = create_random_csv_object(num_of_rows,10)
-    upload_csv_object(bucket_name,csv_obj_name,obj_to_load)
+    upload_object(bucket_name,csv_obj_name,obj_to_load)
     res = remove_xml_tags_from_result( run_s3select(bucket_name,csv_obj_name,"select count(0) from s3object;") ).replace(",","")
 
     s3select_assert_result( num_of_rows, int( res ))
 
+@pytest.mark.s3select
+def test_count_json_operation():
+    json_obj_name = get_random_string()
+    bucket_name = "test"
+    num_of_rows = 1
```

Comment:
it makes sense to have much more rows (> 1). 
for one test it returns the number of rows 
for the other, it returns 1